builder/core/data.py

```diff
@@ -555,7 +555,10 @@ class PKG_TOOLS(Enum):
         'yum_compiler_packages': ['gcc', 'gcc-c++'],
 
         'versions': {
-            '4.8': {},
+            '4.8': {
+                'cmake_args': [
```

Comment:
dont think, this is the right place. we have separate flags for aws-lc https://github.com/awslabs/aws-crt-builder/blob/a31bfacf48d2d10497397674c9928238531b7a50/builder/imports/awslc.py

Reply:
but that should be a gcc4.8 only flag, right?

Reply:
it is. this is aws-lc only flag so arguably we should not be passing it to everything that builds with 4.8. 
ideally we would specify specifically for aws-lc that it should build with that flag on 4.8, but lc does not support builder. file i linked above is where aws-lc specific config gets added, we should be able to add compiler specific cmake there

Reply:
Yep, that's better, updated